packages/create-nuxt-app/lib/prompts.js

```diff
@@ -112,5 +112,15 @@ module.exports = [
       { name: 'Semantic Pull Requests', value: 'semantic-pull-requests' }
     ],
     default: []
+  },
+  {
+    name: 'devOps',
+    message: 'DevOps tools:',
+    type: 'list',
+    choices: [
+      { name: 'None', value: 'none' },
+      { name: 'Dockerize project', value: 'docker' }
```

Comment:
Can be simplify `Docker` or probably question asked as an option when setting target? Deployments are usually not known when initiating a project.

Reply:
I changed its name to `Docker`.